src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AssessmentRepository.java

```diff
@@ -2,18 +2,15 @@
 
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
-import uk.gov.justice.hmpps.prison.api.model.Assessment;
 import uk.gov.justice.hmpps.prison.repository.jpa.model.AssessmentEntry;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderAssessment;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface AssessmentRepository extends CrudRepository<AssessmentEntry, Long> {
     @Query("SELECT question FROM AssessmentEntry question " +
         "INNER JOIN question.parentAssessment questionSet INNER JOIN questionSet.parentAssessment type " +
         "WHERE type.assessmentId = :assessmentId AND type.parentAssessment is null " +
-        "AND type.cellSharingAlertFlag = 'Y' AND questionSet.assessmentCode != 'COMPLETE' " +
+        "AND type.cellSharingAlertFlag = 'Y' AND questionSet.assessmentCode <> 'COMPLETE' " +
```

Comment:
Work around bug in latest Spring data which removed `!=` alias allowed by Hibernate - fixed in version 3.2 which has not been released yet.